js/plugins/checks/README.md

```diff
@@ -1,12 +1,14 @@
 # Checks
 
-Checks is an AI safety platform built by Google: [checks.google.com/ai-safety](https://checks.google.com/ai-safety).
+**Checks** is an AI safety platform built by Google: [checks.google.com/ai-safety](https://checks.google.com/ai-safety).
 
-This plugin provides evaluators for each Checks AI safety policy. Text is cassified by calling the [Checks Guardrails API](https://console.cloud.google.com/marketplace/product/google/checks.googleapis.com).
+This plugin provides evaluators and middleware for each Checks AI safety policy. Text is classified by calling the [Checks Guardrails API](https://console.cloud.google.com/marketplace/product/google/checks.googleapis.com).
```

Comment:
Could we update the URL to be `https://console.cloud.google.com/marketplace/product/google/checks.googleapis.com?project=_` so it brings up the project selector first?

Reply:
done!